tests/Equinox.MessageDb.Integration/MessageDbIntegration.fs

```diff
@@ -0,0 +1,236 @@
+module Equinox.Store.Integration.MessageDbIntegration
```

Comment:
ideally this entire file doesnt get forked (I know #ifdefs are a curse but it would be great to be able to see that messagedb rules out compared to the other sql stores if that's not a completely crazy mess)

Reply:
I think it's a mess, but not a "crazy mess." I can live with it 😄

Reply:
there are definitely many things that are almost at the edge of needing a big cleanup now; just as well I didnt try to be too clever though; def would not have predicted messagedb or dynamo a short time ago!